packages/widgetbook_exception/lib/src/widgetbook_exception.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022, Jens Horstmann
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+/// {@template widgetbook_exception}
+/// An exception thrown by an internal widgetbook command.
+/// {@endtemplate}
+class WidgetbookException implements Exception {
```

Comment:
This class should be abstract to make sure it's not initialized anywhere. This forces developers to create re-usable Exceptions designed for a specific error case.